resource-aggregate/cqrs/eventstore/mongodb/eventstore.go

```diff
@@ -270,6 +277,18 @@ func makeDBETag(etag *eventstore.ETagData) bson.M {
 	}
 }
 
+func tryToSetServiceID(doc bson.M, events []eventstore.Event) bson.M {
+	var serviceID string
+	var ok bool
+	for _, e := range events {
+		serviceID, ok = e.ServiceID()
```

Comment:
This code is intended to get the serviceID of the last event? If so I would expect something like this instead of a for cycle:

```go
e, ok := events[len(events-1)].ServiceID()
```

Reply:
fixed